src/site/content/en/blog/hid/index.md

```diff
@@ -170,9 +169,9 @@ const [device] = await navigator.hid.requestDevice({ filters });
 const devices = await navigator.hid.getDevices();
 ```
 
-<figure>
+<figure class="w-figure">
```

Comment:
This seems to be putting back the old mark-up. With the new design system we don't need these classes.

Reply:
This is a wrong. Sorry for the noise.